5.5.md

```diff
@@ -0,0 +1,3 @@
+Как пользователь имеющий абсолютные права доступа к документу,
```

Comment:
What is "absolute"? It was not defined earlier.

Reply:
This is the definition of this access group. "absolute" is just a name